src/redux/session.js

```diff
@@ -99,20 +99,20 @@ const handleSessionResponse = (dispatch, body) => {
 
 module.exports.refreshSession = () => (dispatch => {
     dispatch(module.exports.setStatus(module.exports.Status.FETCHING));
-    return new Promise((resolve, reject) => (
-        requestSession(resolve, reject).then(body => {
-            handleSessionResponse(dispatch, body);
-        }, err => {
-            dispatch(module.exports.setSessionError(err));
-        })
-    ));
+    return new Promise((resolve, reject) => {
+        requestSession(resolve, reject);
+    }).then(body => {
+        handleSessionResponse(dispatch, body);
+    }, err => {
+        dispatch(module.exports.setSessionError(err));
+    });
 });
 
 module.exports.refreshSessionWithRetry = () => (dispatch => {
     dispatch(module.exports.setStatus(module.exports.Status.FETCHING));
-    return new Promise((resolve, reject) => (
-        requestSessionWithRetry(resolve, reject, 4, 7500)
-    )).then(body => {
+    return new Promise((resolve, reject) => {
+        requestSessionWithRetry(resolve, reject, 4, 7500);
+    }).then(body => {
```

Comment:
What was the error you were getting before? Was it  both here and above? In this case you were returning the call to requestSessionWithRetry as an object literal? At least that's what I understand the () syntax to mean vs {}.  Are they functionally any different right here? I like this better, I just want to make sure I understand the root of the actual problem.

Reply:
We spoke about this in person. @picklesrus pointed out that these lines' change from () to {} don't change functionality; they only clarify that the return value of `requestSessionWithRetry` doesn't matter.